packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -280,26 +281,27 @@ function OrgNav({
       {!disabled && !isMobile && (
         <>
           <OrgNavLink
-            href={current && orgNav.accessManager}
             isActive={activeNav === ActiveNavElement.OrgNavAccessManager}
             loading={!current}
             data-testid="org-nav-access-manager"
-            onClick={onElementClick(NavElement.OrgNavAccessManager)}
-          >
-            Access Manager
-          </OrgNavLink>
-
-          <IconButton
-            aria-label="Dropdown"
-            active={accessManagerOpen}
-            disabled={!current}
-            data-testid="org-nav-dropdown"
-            onClick={onElementClick(NavElement.OrgNavDropdown, () =>
+            onClick={onElementClick(NavElement.OrgNavAccessManager, () =>
```

Comment:
If we're going to make it a breaking change I feel like `OrgNavAccessManagerDropdown` is the most clear name.

Reply:
👍